Mathlib/RingTheory/DiscreteValuationRing/TFAE.lean

```diff
@@ -149,87 +150,41 @@ theorem maximalIdeal_isPrincipal_of_isDedekindDomain [LocalRing R] [IsDomain R]
     · rwa [Submodule.span_le, Set.singleton_subset_iff]
 #align maximal_ideal_is_principal_of_is_dedekind_domain maximalIdeal_isPrincipal_of_isDedekindDomain
 
-theorem DiscreteValuationRing.TFAE [IsNoetherianRing R] [LocalRing R] [IsDomain R]
-    (h : ¬IsField R) :
+/--
+The following are equivalent for a noetherian local domain `(R, m, k)`:
```

Comment:
I implicitly assumed that `R` could not be a field, in this statement, until I read the end of this docstring. How about
```suggestion
Let `(R, m, k)` be a noetherian local domain (possibly a field).
The following are equivalent:
```